src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.ts

```diff
@@ -153,6 +155,8 @@ export class GrafanaComponent implements OnInit, OnChanges {
       four: 'grafana_four'
     };
 
+    this.type === 'metrics' ? (this.datasource = 'Dashboard1') : (this.datasource = 'Loki');
```

Comment:
Don't you like more like this?:
```suggestion
    this.datasource = (this.type == 'metrics') ? 'Dashboard1' : 'Loki';
```